### PR TITLE
Add connector exporter when creating a connector

### PR DESCRIPTION
**Pull request by imsuneth**
Improvement https://github.com/wso2/api-manager/issues/1830

### CODE REVIEW

File: components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/servlets/ConnectorDownloadServlet.java [hunk 1]
```diff
@@ -89,15 +86,19 @@ protected IStatus run(IProgressMonitor monitor) {
...
                                     }
                                 } catch (ConnectorException e) {
                                     WizardDialogUtils.showErrorMessage(e.getMessage(), DOWNLOADING_CONNECTOR_FAILED_MSG);
+                                    
                                 }
                                 monitor.worked(100);
                                 monitor.done();
                                 return Status.CANCEL_STATUS;
                             }
+
```
Comment (rosensilva) [inline components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/servlets/ConnectorDownloadServlet.java:100]: shall we remove the additional new lines?

File: components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/ui/wizard/CloudConnectorImportWizard.java [hunk 2]
```diff
@@ -86,9 +93,14 @@ public boolean performFinish() {
             }
         } else if (removeWizardPage.equals(getContainer().getCurrentPage())) {
             return performFinishRemove();
+        } else if (connectorExporterWizardPage.equals(getContainer().getCurrentPage())) {
+            return performFinishAddToExporter();
         }
         return true;
     }
+
```
Comment (rosensilva) [inline components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/ui/wizard/CloudConnectorImportWizard.java:101]: shall we remove the additional new lines?

File: components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/servlets/ConnectorDownloadServlet.java (hunk 1 above)
Comment (imsuneth) [reply components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/servlets/ConnectorDownloadServlet.java:98]: fixed https://github.com/wso2/integration-studio/pull/1225/commits/32c29dcb1b7672b6d4b7e47378aa9862f28bf37c

File: components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/ui/wizard/CloudConnectorImportWizard.java (hunk 2 above)
Comment (imsuneth) [reply components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/ui/wizard/CloudConnectorImportWizard.java]: fixed https://github.com/wso2/integration-studio/pull/1225/commits/32c29dcb1b7672b6d4b7e47378aa9862f28bf37c

**Review by rosensilva — APPROVED** [review]

**Pull request merged**